README.md

```diff
@@ -6,6 +6,8 @@ Collection of KUDO operators.
 
 The official repository is hosted on Google Cloud Storage under the [Kudo Repository](https://kudo-repository.storage.googleapis.com) bucket.
 
+### [Apache Cassandra](https://github.com/kudobuilder/operators/tree/master/repository/cassandra)
```

Comment:
Please add a one-sentence description like we have for the other operators.

Reply:
5549186267f11a59ca4921caefedfe64e5833989